### PR TITLE
Add support for Turkish language

### PR DESCRIPTION
**Pull request by gnardini**
### Description

Add Turkish to the app.

Note that there are many changes to the format of fonts. I don't know why those were changed :( Does anyone have any ideas?

### Other changes

N/A

### Tested

Manually

### How others should test

Set the Turkish language and check that it works :)

### Related issues

- Fixes #

### Backwards compatibility

N/A

### CODE REVIEW

File: packages/mobile/ios/celo.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -8,9 +8,9 @@
 
 /* Begin PBXBuildFile section */
 		00E356F31AD99517003FC87E /* celoTests.m in Sources */ = {isa = PBXBuildFile; fileRef = 00E356F21AD99517003FC87E /* celoTests.m */; };
-		0317E786245B696200AAAC2E /* Inter-Regular.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 0317E783245B696200AAAC2E /* Inter-Regular.ttf */; };
-		0317E788245B696200AAAC2E /* Inter-Medium.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 0317E784245B696200AAAC2E /* Inter-Medium.ttf */; };
-		0317E78A245B696200AAAC2E /* Inter-SemiBold.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 0317E785245B696200AAAC2E /* Inter-SemiBold.ttf */; };
+		0317E786245B696200AAAC2E /* ../fonts/Inter-Regular.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 0317E783245B696200AAAC2E /* ../fonts/Inter-Regular.ttf */; };
```
Comment (kathaypacific) [inline packages/mobile/ios/celo.xcodeproj/project.pbxproj:11]: i thought i was the only one getting these fonts changes sometimes! nice :D

**Review by kathaypacific — APPROVED** [review]
🇹🇷

**Comment by ValoraQA** [comment]
Hey @MuckT we would cover the turkish language while performing localisation testing and share the doc link here 
Thanks!!!

**Comment by ValoraQA** [comment]
Hey @MuckT we have verified the above task on latest Android Internal release build and latest iOS Testflight release build V:1.28.1 and updated the observations in the Gsheet 

Gsheet Link : [Link](https://docs.google.com/spreadsheets/d/1xD_2Hnbddy8rNJaii8pUD7cl4qDa8256feDmPzYYdOw/edit#gid=0)

CC : @gnardini 
Thanks ,!!

**Pull request merged**